test/Elastic.Apm.Tests/Elastic.Apm.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>

```

Comment:
Hmm.. this would mean that we only test under 2.2. Ideally we should test under every possible framework. The plan here is to multitarget here.

Reply:
I've reverted. If this affects my client project, I'll let you know tomorrow (Friday) when I test.